modules/microshift-security-context-constraints-alert-eval.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * microshift_running_apps/microshift-authentication.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations 
+
+The PodSecurityViolation alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find security violations.
+//not sure if that last bit is accurate 
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
```

Comment:
@copejon Thank you for reviewing! I'm taking a look at that bug as well. I just updated this PR to based on your suggestions for this doc. How does it look so far?